client/components/contact/ContactImage.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import './styles/contactimage.scss';
+
+const ContactImage = ({ children }) => (
```

Comment:
Is `children` the right name here? IIRC, `children` is being set to "Contact Us" in Contact.jsx.